vagrant/provisioning/roles/arkcase-app/templates/arkcase-commons-server.yaml

```diff
@@ -22,7 +22,6 @@ spring:
     mongodb:
       port: "{{ mongodb_port | default(27017) }}"
       host: "{{ file_management_mongodb_host | default(internal_host) }}"
-      uri: "mongodb://{{ file_management_mongodb_host | default(internal_host) }}:{{ mongodb_port | default(27017) }}/file_management_mongodb_database"
```

Comment:
I'm pretty sure the entire `mongodb` block was removed from arkcase-commons.yaml in the .arkcase repository... if so, the whole block should be removed from here too.